Corona-Warn-App/config/detekt.yml

```diff
@@ -522,7 +522,8 @@ style:
       '**/*.Spek.kt',
       '**/CwaWebException.kt',
       '**/HttpErrorParser.kt',
-      '**/ErrorCodes.kt'
+      '**/ErrorCodes.kt',
+      '**/submission/TransmissionRiskVectorDeterminator.kt'
```

Comment:
Why is this necessary?

Reply:
@AlexanderAlferov

Reply:
because transmissionriskvector is allowed to have magic umbers

Reply:
For me okay here, since we have a LOT of magic Numbers otherwise that are just set numbers in the Vector.